docker/test/fuzzer/run-fuzzer.sh

```diff
@@ -345,7 +345,7 @@ quit
         echo "failure" > status.txt
         { rg -ao "Found error:.*" fuzzer.log \
             || rg -ao "Exception:.*" fuzzer.log \
-            || echo "Fuzzer failed ($fuzzer_exit_code). See the logs." ; } \
+            || echo "Fuzzer fucked up with error code: ($fuzzer_exit_code). Its process died somehow when the server stayed alive. The server log probably won't tell you much so try to find information in other files." ; } \
```

Comment:
The problem is, if I'm not mistaken, that this message only gets put into description if `rg` doesn't find an exception. Since the fuzzer.log is filled with expected exceptions, we get the first random one. I'd rather get this message all the time.

Reply:
I guess the wording needs improvement (no profanity) as mentioned in https://github.com/ClickHouse/ClickHouse/pull/53072